Userland/Libraries/LibCore/MimeData.cpp

```diff
@@ -72,11 +72,49 @@ String guess_mime_type_based_on_filename(const StringView& path)
         return "text/markdown";
     if (path.ends_with(".html", CaseSensitivity::CaseInsensitive) || path.ends_with(".htm", CaseSensitivity::CaseInsensitive))
         return "text/html";
+    if (path.ends_with(".js", CaseSensitivity::CaseInsensitive))
+        return "application/javascript";
+    if (path.ends_with(".json", CaseSensitivity::CaseInsensitive))
+        return "application/json";
+    if (path.ends_with(".md", CaseSensitivity::CaseInsensitive))
+        return "text/markdown";
     if (path.ends_with("/", CaseSensitivity::CaseInsensitive))
         return "text/html";
     if (path.ends_with(".csv", CaseSensitivity::CaseInsensitive))
         return "text/csv";
     return "text/plain";
 }
 
+#define ENUMERATE_HEADER_CONTENTS(V)                                                                   \
```

Comment:
These usually follow the pattern (with some exceptions)

```
#define ENUMERATE_FOOBARS   \
    __ENUMERATE_FOOBAR(...) \
    __ENUMERATE_FOOBAR(...) \
    ...
```

Can we do the same here? `V` is not a great name :) Maybe something like `ENUMERATE_MIME_TYPE_HEADERS` / `__ENUMERATE_MIME_TYPE_HEADER`.

Reply:
It sure isn't! I just wanted to keep the lines shorter. I'll change it.